x/evm/spec/04_transactions.md

```diff
@@ -14,7 +14,7 @@ An EVM state transition can be achieved by using the `MsgEthereumTx`. This messa
 type MsgEthereumTx struct {
  // inner transaction data
  Data *types.Any `protobuf:"bytes,1,opt,name=data,proto3" json:"data,omitempty"`
- // encoded storage size of the transaction
+ // DEPRECATED: encoded storage size of the transaction
  Size_ float64 `protobuf:"fixed64,2,opt,name=size,proto3" json:"-"`
```

Comment:
why not use `reserved`?

Reply:
Because it'll break the tx format, not sure if we want to do a breaking change like that.